dftimewolf/cli/dftimewolf_recipes.py

```diff
@@ -115,11 +115,22 @@ def main():
     collector_cls = config.Config.get_collector(collector['name'])
     collector_objects.extend(collector_cls.launch_collector(**new_args))
 
+  # Set global errors for final reporting
```

Comment:
Please fix this comment to describe this better - what's a global error? It's not being set to anything here, either.
And all comments need to end with a period, question mark, or exclamation point.